src/Http/Cookie/Cookie.php

```diff
@@ -440,9 +446,11 @@ public function withExpired()
      */
     public function check($path)
     {
-        $this->_isExpanded();
```

Comment:
I might be wrong but we are missing a `$value = $this->value;` before the `if` because `Hash:::check()` expect an array as the first parameter but `$value` could be `null`here.